src/helpers/services/postNoteService.js

```diff
@@ -0,0 +1,19 @@
+import axios from 'axios';
+
+const postNoteService = async(notesDispatch, note, setNote) => {
+    const encodedToken = "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJfaWQiOiI4ZTc5YTI1Ni01ZmQ2LTQwZTktYmYwYS04NGUyNGE5MDNmYzAiLCJlbWFpbCI6ImFkYXJzaGJhbGlrYUBnbWFpbC5jb20ifQ.tPP0Mb50-2ihtbeWd7PUpf1_pkzVQjmcxNVBIuqzFlU"
+
+    const Headers = { authorization: encodedToken}
+
+    const noteToSend = {note : note}
+    try {
+        const res = await axios.post("/api/notes", noteToSend, {headers: Headers});
+        console.log(res.status, 'post')
+        console.log(res.data.notes, 'post')
```

Comment:
Always remove console logs before pushing the code

Reply:
thanks for all inputs.